src/reducers.js

```diff
@@ -59,11 +57,14 @@ handlers.onComplete = function (state, action) {
  * @private
  */
 export default function createReducer(apiName) {
+  const reApiAction = new RegExp(`^${apiName}_(RESET|START|SUCCESS|FAIL)$`);
```

Comment:
Could `apiName` contain non-regex-safe characters? Might want to do some escape sequences. This might be a good guide for that: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#Escaping

Reply:
Good point, better safe than sorry. Fixed in fbac4d7